### ISSUE TITLE
Fluent Pages version 1.1 fails unit tests

### ISSUE THREAD

**Issue by jmurty** [ic-labs/django-icekit#194]
The recent [1.1](https://github.com/django-fluent/django-fluent-pages/blob/v1.1/CHANGES.rst) release of *django-fluent-pages* causes ICEkit's unit tests, and probably the project in general, to fail with the error:
```
  File "/usr/local/lib/python2.7/dist-packages/fluent_pages/adminui/pageadmin.py", line 135, in _select_template_name
    template_name = native_str(template_name)  # consistent value for lazy() function.
NameError: global name 'native_str' is not defined
```

**Comment by jmurty** [comment]
Raised issue against fluent-pages: https://github.com/django-fluent/django-fluent-pages/issues/125

**Comment by jmurty** [comment]
I have submitted a pull request against the upstream _django-fluent-pages_ project with a simple import fix for this issue: https://github.com/django-fluent/django-fluent-pages/pull/126

**Comment by jmurty** [comment]
The django-fluent-pages issue is fixed in version 1.1.1
